olsync/olsync.py

```diff
@@ -17,53 +17,60 @@
 import zipfile
 import io
 import dateutil.parser
+import glob
+import fnmatch
 
 
 @click.group(invoke_without_command=True)
 @click.option('-l', '--local-only', 'local', is_flag=True, help="Sync local project files to Overleaf only.")
 @click.option('-r', '--remote-only', 'remote', is_flag=True,
               help="Sync remote project files from Overleaf to local file system only.")
 @click.option('--store-path', 'cookie_path', default=".olauth", type=click.Path(exists=False),
-              help="Path to load the persisted Overleaf cookie.")
+              help="Relative path to load the persisted Overleaf cookie.")
 @click.option('-p', '--path', 'sync_path', default=".", type=click.Path(exists=True),
               help="Path of the project to sync.")
+@click.option('--olignore', 'olignore_path', default=".olignore", type=click.Path(exists=False),
```

Comment:
What's the reasoning for the `Path(exists=False)`? Shouldn't the `.olignore` file be there if specified?

Reply:
Right, it should be `exists=True`.

I guess what I want to do is: if `.olignore` file does not exist, it is alright (ignore no item); if specified, it should be a valid `.olignore`-like (maybe different name) file.